OpenRA.Mods.Cnc/Traits/SupportPowers/ChronoshiftPower.cs

```diff
@@ -249,11 +244,28 @@ IEnumerable<Order> OrderInner(CPos xy)
 			{
 				// Cannot chronoshift into unexplored location
 				if (IsValidTarget(xy))
-					yield return new Order(order, manager.Self, Target.FromCell(manager.Self.World, xy), false)
+				{
+					var units = power.UnitsInRange(sourceLocation).Where(a => IsValidTarget(xy, a, manager.Self.Owner));
+
+					yield return new Order(order, manager.Self, Target.FromCell(manager.Self.World, xy), false, units.ToArray())
```

Comment:
You forgot to check for `units.Any()` before sending the order. (?)

Reply:
Yeah, sorry. Fixed.